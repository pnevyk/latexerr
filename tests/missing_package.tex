\documentclass{article}

\usepackage{missing}

\begin{document}
\end{document}
