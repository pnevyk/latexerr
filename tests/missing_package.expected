File: missing_package.tex

Error: Missing package missing.
